apps/server/src/shared/domain/entity/news.entity.ts

```diff
@@ -1,23 +1,24 @@
 import { Entity, Enum, ManyToOne, Property } from '@mikro-orm/core';
-import { BaseEntityWithTimestamps, EntityId } from '@shared/domain';
-import { CourseInfo } from './course-info.entity';
-import { NewsTargetInfo, NewsTargetModel } from './news.types';
-import { SchoolInfo } from './school-info.entity';
-import { TeamInfo } from './team-info.entity';
-import { UserInfo } from './user-info.entity';
+import { BaseEntityWithTimestamps } from './base.entity';
+import type { Course } from './course.entity';
+import type { School } from './school.entity';
+import type { Team } from './team.entity';
+import type { User } from './user.entity';
+import { NewsTarget, NewsTargetModel } from '../types/news.types';
+import { EntityId } from '../types';
 
 export interface INewsProperties {
 	title: string;
 	content: string;
 	displayAt: Date;
-	school: EntityId;
-	creator: EntityId;
-	target: EntityId;
+	school: EntityId | School;
```

Comment:
Should it possible to add only the entityId for creating?

Reply:
actually no. we should use the real entities in the future to ensure referential integrity. but for that we need the repositories for `School`, `Team` and `Course`. these are not yet all available. adding them would be maybe more in the scope of another ticket.